rook-to-koor.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# script for resolving patching rook to koor changes, this uses setting up a
+# worktree with latest rook/master and mangling it to adopt koor conventions,
+# creating a temporary commit.
+
+set -xe
+
+KOORDIR=$(pwd)
+ROOKDIR=$(pwd)/temp_rook_merge
+
+#setup a temporary rook latest folder
+git fetch rook master
+git worktree add ${ROOKDIR} --checkout rook/master
+cd ${ROOKDIR}
+git checkout -b wip-temp-rook-merge
+
+# ordering matters here
+find ${ROOKDIR} -type f ! -path "./.git/*" -exec sed -i -e 's|rook/rook.github.io|koor-tech/docs.koor.tech|g' \
+-e 's|rook/rook/issues|koor-tech/koor/issues|g' \
+-e 's|/work/rook/rook|/work/koor/koor|g' \
+-e 's|/work/rook|/work/koor|g' \
+-e 's|rook/rook|koor/koor-tech|g' \
+-e 's|[Rook.io Slack](https://slack.rook.io/)|[GitHub Discussions](https://github.com/koor-tech/koor/discussions)|g' \
+-e 's|Rook version|Koor Storage Distribution version|g' \
+-e 's|rook.io/docs/rook|docs.koor.tech/docs|g' \
+-e 's|"$GITHUB_REPOSITORY_OWNER" = "rook"|"$GITHUB_REPOSITORY_OWNER" = "koor-tech"|g' \
+-e 's|rook/ceph|koorinc/ceph|g' \
+-e 's|rook.chart|koor.chart|g' \
+-e 's|version of Rook|version of Koor Storage Distribution|g' \
+-e 's/Rook*$ website/Koor website/g' \
+-e 's|rook-release|koor-release|g' \
+-e 's|charts.rook.io|charts.koor.tech|g' \
+-e 's|Rook Ceph Operator|Koor Operator|g' \
+-e 's|cncf-rook-security@lists.cncf.io|security@koor.tech|g' \
+-e 's|cncf-rook-distributors-announce@lists.cncf.io|the Koor Newletter|g' \
+-e 's|rook_io|koor_tech|g' \
+-e 's|Rook Ceph |Koor |g' \
+-e 's/https\:\/\/slack\.rook\.io.*/\[the GitHub Discussions\]\(https\:\/\/github\.com\/koor\-tech\/koor\/discussions\)/g' \
+-e 's|src/github.com/rook|src/github.com/koor-tech|g' {} \;
+
+# places where complete replacement are apt
+sed -i "s|Rook|Koor Storage Distribution|g" ${ROOKDIR}/.github/PULL_REQUEST_TEMPLATE.md \
+${ROOKDIR}/CODE_OF_CONDUCT.md \
+${ROOKDIR}/Documentation/Contributing/development-flow.md \
+${ROOKDIR}/README.md
+
+# release version mangling 1.10 and 1.9 -> 1.0
+sed -i "s/release\-[0-9].[0-9]/release\-1\.0/g" ${ROOKDIR}/Documentation/Contributing/development-flow.md
+sed -i 's/v1\.10\../1\.0\.0/g' ${ROOKDIR}/Documentation/Upgrade/rook-upgrade.md
```

Comment:
Maybe in the future we can use some `git` commands to get the latest tag of Koor, not needed for this PR.